src/components/Room.tsx

```diff
@@ -47,7 +52,8 @@ function Room() {
   useEffect(() => {
     console.log("timestamp", timestamp);
     // update the document title, with roomName and timestamp
-    document.title = `${timestamp}-${window.location.href.split("/")[3]}`;
+    const formattedTimestamp = formatTimestamp(timestamp);
+    document.title = `${formattedTimestamp}-${window.location.href.split("/")[3]}`;
```

Comment:
Instead of adding 2 lines you can just combine it into a single line
```suggestion
    document.title = `${formatTimestamp(timestamp)}-${window.location.href.split("/")[3]}`;
```